lib/krane/cli/krane.rb

```diff
@@ -17,6 +17,11 @@ class Krane < Thor
 
       package_name "Krane"
 
+      def self.start(*)
+        Oj.mimic_JSON
+        super
+      end
+
```

Comment:
Since this file is only required by the CLI binary, there should be no problems with leaking to the requiring context. So I think we can just do:
```
require 'oj'
Oj.mimic_JSON
```
at the top of this file. I don't think we need to use the `start` call for this.